test/verify/machineslib.py

```diff
@@ -738,11 +738,20 @@ def testNetworks(self):
         b.click("tbody tr[data-row-id=vm-subVmTest1] th") # click on the row header
         b.wait_in_text("#vm-subVmTest1-state", "running")
 
+        # Wait for the dynamic IP address to be assigned before logging in
+        # If the IP will change or get assigned after fetching the domain data the user will not see any
+        # changes until they refresh the page, since there is not any signal associated with this change
+        wait(lambda: "1" in self.machine.execute("virsh domifaddr subVmTest1  | grep 192.168.122. | wc -l"), delay=3)
```

Comment:
This is a rather strange check. grep already returns a sensible error code, so just

    wait(lambda: self.machine.execute("virsh domifaddr subVmTest1  | grep 192.168.122."), delay=3)

should do fine.